main.cpp

```diff
@@ -315,41 +329,166 @@ void showShop (){
             }
         }
 
-        shop.clear(Color::White);
-
+        shop.clear(Color::Cyan);
         if (Mouse::isButtonPressed(Mouse::Left)){
             Vector2i MousePos = Mouse::getPosition(shop);
-            for (auto button = 0; button < opButtons.size(); button++){
-                if (opButtons[button].getGlobalBounds().contains(MousePos.x, MousePos.y)){ 
-
+            for (auto button = 0; button < products.size(); button++){
+                if (plusButton[button].getGlobalBounds().contains(MousePos.x, MousePos.y)){ 
                     order = editOrder(order, button, goods);
-                    goods = updateGoods();
 
-                    for (int i = 0; i < order.name.size(); i++){
-                        cout << order.name[i] << endl << order.count[i] << endl;
-                    }
-                    for (auto i = 0; i < goods.name.size(); i++){
-                        countProd[i].setString(goods.count[i]);
-                    }
+                    stock[button] = editOne(stock[button], false, goods.count[button]);
+                    stockText[button].setString(stock[button]);
+
+                    orderGoodName = getDataGood(order);
+                    orderGoodCount = getDataCount(order);
+
+                    minButton = setDataButton(order);
+                    minusText = setDataText(order);
+                }
+            }
+            for (auto button = 0; button < order.name.size(); button++){
+                if (minButton[button].getGlobalBounds().contains(MousePos.x, MousePos.y)){ 
+                    int index = stockName(order.name[button], goods);
+                    order = editOrder(order, button, goods, true);
+
+                    stock[index] = editOne(stock[index], true, goods.count[index]);
+                    stockText[index].setString(stock[index]);
+
+                    orderGoodName = getDataGood(order);
+                    orderGoodCount = getDataCount(order);
+
+                    minButton = setDataButton(order);
```

Comment:
minButton -> minusButton. Don't use too short words that can be interpreter like other words (min is minimum)